src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -185,7 +185,7 @@ public String FullName
             private set
             {
                 if (value == null)
-                    throw new ArgumentNullException("FullName");
+                    throw new ArgumentNullException(nameof(FullName));
```

Comment:
No, let's not make such changes. It's true there are inconsistencies across corefx over whether the property name or "value" is used, but until we decide to standardize on one or the other, let's not churn these, especially since it is a visible change.